src/views/Users/ViewUser/index.jsx

```diff
@@ -289,15 +393,27 @@ function ViewUser({ isNewUser, ...props }) {
       <Grid item xs={4}>
         <AutoCompleteText
           multi
-          disabled={!supportsActionRestriction(permission.name)}
-          selectedItems={permission.options.actions}
+          disabled={
+            (permission.sc && permission.sc.change_type === 'delete') ||
+            !supportsActionRestriction(
+              permission.sc ? permission.sc.name : permission.name
+            )
+          }
+          selectedItems={
+            permission.sc
+              ? propOr([], 'actions')(permission.sc.options)
+              : permission.options.actions
+          }
           onSelectedItemsChange={handleRestrictionChange(permission, 'actions')}
           onValueChange={handleRestrictionTextChange(permission, 'actionText')}
           value={permission.metadata.actionText}
           getSuggestions={getSuggestions(getSupportedActions(permission.name))}
           label="Action Restrictions"
         />
       </Grid>
+      {/* TODO: This delete button functions weirdly when a permission
+          is scheduled to be deleted. If used, it will _cancel_ the
+          deletion. Need to find a better way to handle this. */}
```

Comment:
We should probably have an issue filed for this.

Reply:
I filed https://github.com/mozilla-frontend-infra/balrog-ui/issues/106 for it.